kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -316,6 +316,21 @@ def run(self, labels: list, analysis_id=None):
                     runtime_environments = [
                         e["name"] for e in thoth_config["runtime_environments"]
                     ]
+                    managers = thoth_config.get("managers")
+                    for mgr in managers:
+                        if mgr.get("name") == "thoth-advise":
+                            disallow_run_times = mgr.get("disallow", None)
+                            allowed_run_times = mgr.get("allow", None)
```

Comment:
This shouldn't have to be done here, should be passed through the managers configuration variables.